README.md

```diff
@@ -49,11 +49,67 @@ If a function should never be called from another contract, it should be marked
 
 #### 1. Errors
 
-##### A. Prefer custom errors.
+##### A. Using Custom Errors Over Require
 
-Custom errors are in some cases more gas efficient and allow passing useful information.
+Utilize custom errors instead of `require` statements for clearer and more gas-efficient error handling. Solidity 0.8.26 supports the use of custom errors with `require`.
 
-##### B. Custom error names should be CapWords style.
+```solidity
+error InsufficientFunds(uint256 requested, uint256 available);
+
+function withdraw(uint256 amount) public {
+    if (amount > balance) {
+        revert InsufficientFunds(amount, balance);
+    }
+    balance -= amount;
+}
+```
+
+> [!TIP]
+> 💡 Custom errors save gas and provide more detailed error messages compared to traditional `require` strings.
+
+##### B. Require with Custom Error (Solidity 0.8.26+)
```

Comment:
I think this could be another "YES" example with note about 0.8.26

Reply:
Updated